### PR TITLE
Thresholding for Sentinel2A

### LINKED ISSUE #4: Perform Thresholding for Sentinel-2A Images
* Right now, a simple band math formula has been applied to the input image.
* This outputs a grayscale image - the pixels having higher probability of having the feature of interest have a high value (0-255) and hence appear towards the brighter side. 

- [ ] Perform thresholding, decide a threshold value, in order to convert the output grayscale image into a binary feature map

* For eg: If the threshold is decided as 200, then modify the output image such that all pixels having values greater than or equal to 200 will now have a value of 1, and other pixels will have a value of 0. This will result in a binary image, in which the bright pixels would represent features.

### PR DESCRIPTION
**Pull request by BhruguDave**
### Fixes #4

- Performed thresholding for NDWI(leaf moisture content), NDWI(water content), NDCI for Chilikha, Orissa, India

- Performed thresholding for NDWI(leaf moisture content), NDWI(water content), NDCI for Okeechobee, Florida, US

- Updated the readme file accordingly with results as well as algorithm used for thresholding

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,41 @@ For this study, the Landsat 8 images for the same regions were used for which le
...
+
+- Plotted the graph of number of pixels Vs. gray scale to find the value of the threshold
+
+- Generated the binary image with respect to the threshold obtained in the prevoius step
+
+## Results obtained by the implementation of the algorithm
+
+- NDWI(leaf moisture content) for Chilikha, Orissa, India
+
+<img src="misc/thresholding_sentinel2A/chilikha_ndwi_lmc_t144.PNG" width="400">
```
Comment (Chintan2108) [inline README.md]: Image is not showing up in the readme. Same comment for the other four images. Please check.

**Review by Chintan2108 — CHANGES_REQUESTED** [review]
Great work on the code and implementation @BhruguDave!
Before we merge, please check in on the readme, there seems to be some issue - the thresholding results snapshots are not being displayed.

File: README.md (hunk 1 above)
Comment (BhruguDave) [reply README.md]: @Chintan2108 I have fixed the issue with the readme.

**Review by Chintan2108 — APPROVED** [review]

**Comment by Chintan2108** [comment]
give @BhruguDave 75 points

**Pull request merged**